consensus/src/runway/packer.rs

```diff
@@ -0,0 +1,218 @@
+use crate::{
+    units::{FullUnit, PreUnit, SignedUnit},
+    Data, DataProvider, Hasher, MultiKeychain, NodeIndex, Receiver, Sender, SessionId, Signed,
+};
+use futures::{channel::oneshot, pin_mut, FutureExt, StreamExt};
+use log::{debug, error, info};
+use std::marker::PhantomData;
+
```

Comment:
It would be good to write one or two sentences of comment on what this component does.

Reply:
added some comments